pkgs/tools/misc/broot/default.nix

```
@@ -9,27 +9,30 @@
 , oniguruma
 , libiconv
 , Security
+, xorg
 , zlib
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "broot";
-  version = "1.6.6";
+  version = "1.7.1";
 
   src = fetchCrate {
     inherit pname version;
-    sha256 = "sha256-Aezi005CKhGwCRZ2HwxFRRORBvVoZEzljTcKQZ8b8XI=";
+    sha256 = "sha256-Gg60jeDfHNKUYynJRgN4oLWhcdApWUdO5kRwl5hrFMM=";
   };
 
-  cargoHash = "sha256-d06lvS5ajXPY/AwmE2rkya82mAOeEv04EtBElDvX2+I=";
+  cargoHash = "sha256-xsTlFfgX3Np3M/RqmK30DZBCrY1OXnEkjqBDI+ffd/U=";
 
   nativeBuildInputs = [
     installShellFiles
     makeWrapper
     pkg-config
   ];
 
-  buildInputs = [ libgit2 oniguruma ] ++ lib.optionals stdenv.isDarwin [
+  buildInputs = [ libgit2 oniguruma ] ++ (with xorg; [
+    libxcb
+  ]) ++ lib.optionals stdenv.isDarwin [
```

please inherit libxcb inside `all-packages.nix`
Also, are you sure libxcb is needed on darwin too?

```suggestion
  buildInputs = [ libgit2 libxcb oniguruma ] ++ lib.optionals stdenv.isDarwin [
```

---

made both changes: `all-packages.nix` and the respective build input list.